ui/widgets/nuxeo-user-avatar.js

```diff
@@ -296,14 +307,21 @@ import { html } from '@polymer/polymer/lib/utils/html-tag.js';
             '9',
             '0',
           ];
-          let tempName = '';
-          const splitName = name.split(' ');
+
           const alphabetPosition = alphabet.indexOf(name.charAt(0).toLowerCase());
-          this.$.container.style.backgroundColor = colors[alphabetPosition];
-          for (let i = 0; i < splitName.length; i++) {
-            tempName += splitName[i].charAt(0);
+          if (alphabetPosition === -1) {
```

Comment:
nit: I'd save an assignment and do:
```suggestion
          this._isInTheAlphabet = alphabetPosition > -1;
          if (!this._isInTheAlphabet ) {
```
and delete the `this._isInTheAlphabet = true;` below.

Reply:
Much better your way. Integrated.